proto/compliance/tx.proto

```diff
@@ -21,10 +21,10 @@ message MsgCertifyModel {
   int32 vid = 2 [(gogoproto.moretags) = "validate:\"gte=1,lte=65535\""];
   int32 pid = 3 [(gogoproto.moretags) = "validate:\"gte=1,lte=65535\""];
   uint32 software_version = 4;
-  string software_version_string = 5 [(gogoproto.moretags) = "validate:\"required\""];
+  string software_version_string = 5 [(gogoproto.moretags) = "validate:\"required,max=10\""];
```

Comment:
I believe `software_version_string` should be 64 as in ModelVersion and the spec (please fix for all occurrences)